lib/manageiq/appliance_console/message_configuration_server.rb

```diff
@@ -190,6 +199,27 @@ def deactivate_services
         LinuxAdmin::Service.new("zookeeper").stop
         LinuxAdmin::Service.new("kafka").stop
       end
+
+      def assemble_keystore_params
+        keystore_params = {"-keystore" => keystore_path, "-validity" => 10_000, "-genkey" => nil, "-keyalg" => "RSA", "-storepass" => password, "-keypass" => password}
```

Comment:
Super minor, but this might be easier to read if broken up into multple lines.

Reply:
Resolved by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/149/commits/14c07ec105fc32ca6fdf36842456fcde4a54d0a0